dev-libs/libclc/libclc-0.2.0_pre20180610.ebuild

```diff
@@ -31,7 +31,11 @@ fi
 LICENSE="|| ( MIT BSD )"
 SLOT="0"
 KEYWORDS="~amd64 ~arm64 ~x86"
-IUSE=""
+IUSE_VIDEO_CARDS="video_cards_nvidia video_cards_r600 video_cards_radeonsi"
```

Comment:
The extra variable makes no sense here. I suppose you might've wanted to use it in REQUIRED_USE but you didn't.

Reply:
done. I reused the variable